### ISSUE TITLE
[New Feature Request] --transcript-count for counting

### ISSUE THREAD

**Issue by akmazian** [BUStools/bustools#100]
[update]: oops, I just realized you can use `kallisto quant-tcc` to do this

I know there are workarounds for this but it would be nice to have a `--transcript-counts` option for `bustools count` that acts similar as `--genecounts` :)

Rationale:
- for certain purposes, e.g. when studying transcript isoforms, collapsing everything to gene-level loses useful information

Current workaround:
- create a `t2t.txt` file that maps each transcript to itself and use it in place of `t2g.txt`

**Comment by Yenaled** [comment]
yup, just get your TCC matrix (omit --genecounts and put --multimapping to allow reads mapped to multiple genes to be preserved) and then run quant-tcc on the resulting matrix. This is basically the same as running kallisto for bulk RNA-seq (which gets you isoform-level quantification in an accurate probabilistic manner).
